hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/main/java/org/apache/hadoop/yarn/server/router/rmadmin/DefaultRMAdminRequestInterceptor.java

```diff
@@ -80,38 +80,17 @@
   public void init(String userName) {
     super.init(userName);
     try {
-      // Do not create a proxy user if user name matches the user name on
```

Comment:
This part of the initialization code has been implemented in `AbstractRMAdminRequestInterceptor`, we can call `super.init` to resolve the ugi.